glide.lock

```diff
@@ -220,6 +220,11 @@ imports:
   version: f62e98d28ab7ad31d707ba837a966378465c7b57
 - name: github.com/spf13/pflag
   version: 9ff6c6923cfffbcd502984b8e0c80539a94968b7
+- name: github.com/stretchr/testify
```

Comment:
Some of these testify tests have been duplicated in test/util/assertions.go. It's nice to bring in the real testify so that we can get rid of that assertions.go file. I'll write up an issue to do that.

Reply:
Sounds good to me.